release_notes/ocp_3_9_release_notes.adoc

```diff
@@ -0,0 +1,932 @@
+following:
+
+----
+openshift_crio_docker_gc_node_selector={'runtime': 'cri-o'}
+----
+
+For example, the above would ensure it is only run on nodes with the `runtime:
+cri-o` label. This can be helpful if you are running CRI-O only on
+xref:../install_config/build_defaults_overrides.adoc#ansible-setting-global-build-defaults[some
+nodes], and others are only running `docker`.
+
+See the link:http://cri-o.io/[upstream documentation] for more information on
+CRI-O.
+
+[[ocp-39-buildha]]
+==== Buildha
+
+Buildah is fully supported with RHEL 7.5.
+
+Buildah is a daemon-less tool for building and modifying OCI and Docker images.
+It preserves the existing dockerfile workflow and instructions, and allows
+fine-grain control over image layers, the content, and commits. Utilities on the
+container host can optionally be called for the build. Buildah shares the
+underlying image and storage components with CRI-O
+
+[[ocp-39-storage]]
+=== Storage
+
+[[ocp-39-pv-resize]]
+==== PV Resize
+
+You can expand persistent volume claims online from {product-tile} for CNS
+glusterFS, Cinder, and gcePD.
+
+. Create a storage class with  `AllowVolumeExpansion=true`.
+. The PVC uses the storage class and submits a claim.
+. The PVC specifies a new increased size.
+. The underlying PV is resized.
+
+[[ocp-39-end-to-end-online-expansion-resize-for-cns-glusterfs-pvs]]
+==== End-to-end Online Expansion and Resize for CNS Glusterfs PVs
+
+You can expand persistent volume claims online from {product-tile} for CNS
+glusterFS volumes.
+
+This can be done online from {product-title}. Previously, this was only
+available from the Heketi CLI. You edit the PVC with the new size, triggering a
+PV resize This is fully qualified for glusterFs backed PVs. Gluster-block PV
+resize will be added with RHEL 7.5.
+
+. Add `AllowVolumeExpansion=true` to the storage class.
+. Run:
++
+----
+$ oc edit pvc claim-name
+----
+
+. Edit the `storage` field, which is under `spec` and `requests`, with the new value.
+
+[[ocp-container-native-storage-glusterfs-pv-consumption-metrics-available]]
+==== Container Native Storage GlusterFS PV Consumption Metrics Available from {product-title}
+
+Container Native Storage GlusterFS is extended to provide volume metrics
+(including consumption) through Prometheus or Query.
+
+Metrics are available from the PVC endpoint. This adds visibility to what is
+being allocated and what is being consumed. Previously, you could only see
+allocated size of the PVs. Now, you know how much is really consumed so, if
+needed, you can expand it before it runs out of space. This also allows
+administrators to do billing based on consumption, if needed.
+
+Examples of added metrics include:
+
+* `kubelet_volume_stats_capacity_bytes`
+* `kubelet_volume_stats_inodes`
+* `kubelet_volume_stats_inodes_free`
+* `kubelet_volume_stats_inodes_used`
+* `kubelet_volume_stats_used_bytes`
+
+[[ocp-3-9-cns-now-supports-custom-volume-naming]]
```

Comment:
This has not been verified by QE. Sorry but I have no idea where this feature come from. Also it's missing instructions for how to prefix volume name in storageclass.